nengo/tests/test_processes.py

```diff
@@ -99,7 +100,7 @@ def test_gaussian_whitenoise(Simulator, rms, seed, plt):
 def test_whitesignal_rms(Simulator, rms, seed, plt):
     t = 1.
     d = 500
-    process = WhiteSignal(t, rms=rms)
+    process = WhiteSignal(t, high=500, rms=rms)
```

Comment:
Why would you want to remove it? If were were using 3.5 I would force it to be passed as a keyword arg even if it is required; bare numbers and booleans are rarely understandable in an argument list without a keyword or a well-named variable.